webapp/app/routes/singleList/singleList.component.js

```diff
@@ -0,0 +1,144 @@
+
+  componentWillMount() {
+    this.props.fetchList(parseInt(this.props.match.params.id, 10));
+  }
+
+  componentDidMount() {
+    this._timeout = setTimeout(() => this.setState({ headerIn: true }));
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this._timeout);
+  }
+
+  handleBackClick = () => {
+    this.props.history.push(`/${this.props.match.params.lang}`);
+  };
+
+  handleItemClick = (item) => {
+    this.props.toggleItemActive(item.get('id'), !item.get('active'));
+  };
+
+  handleEditClick = () => {
+    // TODO
+  };
+
+  handleDeleteClick = () => {
+    // TODO
+  };
+
+  renderLoader = () => (
+    <LoaderContainer>
+      <CircularProgress size={50} color="secondary" />
+    </LoaderContainer>
+  );
+
+  renderContent = () => {
+    const { list, classes, theme: { transitions: { duration } } } = this.props;
+
+    return (
+      <Container>
+        <Collapse in={this.state.headerIn}>
+          <Typograpghy className={classes.header} variant="title" noWrap>
+            <IconButton color="inherit" aria-label="Menu" onClick={this.handleBackClick}>
+              <HomeIcon />
+            </IconButton>
+
+            <KeyboardArrowRightIcon />
+
+            <TitleContainer>
+              {list.get('name')}
+            </TitleContainer>
+          </Typograpghy>
+          <ListsProgress list={list} />
+        </Collapse>
+
+        <List disablePadding >
+          {list.get('items').toArray().map((item) => (
+            <Fragment key={item.get('id')}>
+              <ListItem
+                role={undefined}
+                dense
+                button
+                onClick={() => this.handleItemClick(item)}
+              >
+                <Checkbox
+                  checked={!item.get('active')}
+                  tabIndex={-1}
+                  disableRipple
+                />
+                <ListItemText disabled={!item.get('active')}>
+                  <Typograpghy variant="body1" color="inherit">
+                    {item.get('name')}
+                  </Typograpghy>
+                </ListItemText>
+                <ListItemSecondaryAction>
+                  <IconButton className={classes.itemIconButton} onClick={this.handleEditClick}>
+                    <EditIcon className={classes.itemIcon} />
+                  </IconButton>
+                  <IconButton className={classes.itemIconButton} onClick={this.handleDeleteClick}>
+                    <DeleteIcon className={classes.itemIcon} />
+                  </IconButton>
+                </ListItemSecondaryAction>
+              </ListItem>
+              <Divider />
+            </Fragment>
+          ))}
+        </List>
+
+        <Zoom in unmountOnExit timeout={{ enter: duration.enteringScreen, exit: duration.leavingScreen }}>
+          <Button variant="fab" color="primary" aria-label="add" className={classes.fab}>
+            <AddIcon />
+          </Button>
+        </Zoom>
+      </Container>
+    );
+  };
+
+  render = () => ifElse(
```

Comment:
`
render = () => ifElse(
    propEq('isLoadingList', true),
    this.renderLoader,
    this.renderContent
 )(this.props);
`
`always` looks weird as the condition